app/models/user.rb

```diff
@@ -8,11 +8,19 @@ class User < ApplicationRecord
   has_many :comments, class_name: 'Comment', foreign_key: 'user_id', dependent: :destroy
```

Comment:
- [ ] Kindly make sure to add `:confirmable` so that the user can confirm their email after signing up. As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation%20of%20email.) requirement.